src/main/java/net/barrage/school/java/ecatalog/config/SecurityConfiguration.java

```diff
@@ -34,11 +40,15 @@ public SecurityFilterChain filterChain(
         http
                 .csrf(Customizer.withDefaults())
```

Comment:
Your POST/DELETE methods return 403. Have u tried it?

Reply:
Yes, It was forbidden but I was adding csrf manually so it worked. I disabled it since my workaround was dumb.

Reply:
ohh, ok, u could leave it disabled actually